operator/gitops/argocd/pipeline-service/openshift-pipelines/pipelines-scc.yaml

```diff
@@ -0,0 +1,37 @@
+---
+apiVersion: security.openshift.io/v1
+kind: SecurityContextConstraints
+metadata:
+  name: pipelines-scc
+allowHostDirVolumePlugin: false
+allowHostIPC: false
+allowHostNetwork: false
+allowHostPID: false
+allowHostPorts: false
+allowPrivilegeEscalation: false
+allowPrivilegedContainer: false
+allowedCapabilities:
+  - SETFCAP
```

Comment:
put a comment that this is an addition from the `restricted` SCC

Reply:
what was the need for this?  .... building images with buildah perhaps ?  whatever it is , clarify that in the comment